### ISSUE TITLE
Custom Noodles revert to default as you explore

### ISSUE THREAD

**Issue by ghost** [Wyn-Price/Noodle#4]
I've been playing with this mod quite a bit, using it as a base for modpack.

If you set noodle type to straight or curly in the customize settings and then explore outside of the spawn area the noodles revert to default as new chunks generate.

The same is also true if you adjust the density.

I've have JourneyMap in my pack and if you look at that you can clearly see the line where it reverts.

**Comment by Wyn-Price** [comment]
By default, do you mean default noodles?

**Comment by ghost** [comment]
yes. The noodle type reverts to the type of noodles created by the default settings.

For example if you set the noodles to straight.. then once you leave that original area... they starte to have curls.
